From: Andrew Morton <akpm@linux-foundation.org>
Subject: kasan-test-use-underlying-string-helpers-checkpatch-fixes

WARNING: Block comments use a trailing */ on a separate line
#49: FILE: lib/test_kasan.c:837:
+ * */

total: 0 errors, 1 warnings, 37 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/kasan-test-use-underlying-string-helpers.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/test_kasan.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/test_kasan.c~kasan-test-use-underlying-string-helpers-checkpatch-fixes
+++ a/lib/test_kasan.c
@@ -834,7 +834,7 @@ static void kmem_cache_invalid_free(stru
 /*
  * noinline wrappers to prevent the compiler from noticing the overflow
  * at compile time rather than having kasan catch it.
- * */
+ */
 static noinline void *__kasan_memchr(const void *s, int c, size_t n)
 {
 	return memchr(s, c, n);
_
